script/zmtest

```diff
@@ -9,11 +9,12 @@ usage () {
     status="$1"
     message="$2"
     printf "%s" "${message}" >&2
-    echo "Usage: zmtest [OPTIONS] DOMAIN" >&2
+    echo "\nUsage: zmtest [OPTIONS] DOMAIN" >&2
```

Comment:
Implementations of `echo` can be very inconsistent in whether they handle options (such as `-n` or `-e`) or backslash-escapes such as `\n`. Instead, it’s better to add a `\n` to the previous `printf`’s format string.

Reply:
Or
```
( echo; echo "Usage: zmtest [OPTIONS] DOMAIN" ) >&2
```